base/linalg/dense.jl

```diff
@@ -509,7 +509,7 @@ function logm(A::StridedMatrix)
         end
     end
 
-    if isreal(A) && ~np_real_eigs
+    if all(isreal, A) && ~np_real_eigs
```

Comment:
I would have thought so, but apparently [`~(x::Bool)` is a synonym for `!x`](https://github.com/JuliaLang/julia/blob/41a81538ce031be8ca8ba882c96dc8507dcf2a8f/base/bool.jl#L40) (`~` being bitwise not)?